### DISCUSSION TITLE
Difference between yarn and pnpm for developing with local version of Next.js

### DISCUSSION THREAD

**Discussion by bytrangle** [vercel/next.js#37369] [Help] [answered]
In the instruction for [developing with local version of Next.js](https://github.com/vercel/next.js/blob/canary/contributing.md#developing-with-your-local-version-of-nextjs), both `yarn` and `pnpm` are used. The former is for installing dependencies, and the latter is for updating dependencies after one makes changes to local Next.js.

 Why are there two package managers in this case and what are the differences between then?

**Comment by ijjk** [comment, marked as answer]
Hi, seems this occurrence of `yarn` just didn't get updated yet, opened PR updating this here https://github.com/vercel/next.js/pull/37370
